xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -20,8 +20,40 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents an assembly load context, which represents the runtime's concept of a load binding scope. This class is abstract and must be implemented to be instantiated.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The <xref:System.Runtime.Loader.AssemblyLoadContext> represents an instance of a load context.  Conceptually, a load context creates a scope for loading, resolving, and potentially unloading a set of assemblies.
+ 
+ <xref:System.Runtime.Loader.AssemblyLoadContext> is an abstract class. While the class is abstract, only the <xref:System.Runtime.Loader.AssemblyLoadContext.Load(System.Reflection.AssemblyName)?displayProperty=nameWithType> needs to be implemented to create a concrete class.
+
+### Usage in the runtime
+ The runtime implements two <xref:System.Runtime.Loader.AssemblyLoadContext>:
+     * The <xref:System.Runtime.Loader.AssemblyLoadContext.Default?displayProperty=nameWithType> represents the runtimes default binding context which is used for the framework and the application main assembly and its normal dependencies as described by its `*.deps.json` file.
```

Comment:
*.deps.json is really just an implementation detail.  I like the focus that Default is the 'runtimes binding context', but I think we should exclude details about deps.json.

Reply:
Done